.github/workflows/testing.yml

```diff
@@ -14,17 +14,12 @@ on:
         required: true
 
 env:
-  PAT: ${{ secrets.PAT }}
```

Comment:
sure that we do not need the PAT anymore? I think that was necessary for the triggering of the docker-build in the other repo?

Reply:
For the workflow-dispatch action that triggers the docker build it is set directly from `secrets.PAT`, not through `env`. See here: https://github.com/Open-MSS/MSS/blob/dc4db0bde011c559bb5ed0eff2530f47876d8a26/.github/workflows/testing.yml#L76-L80

Way back when this was introduced `env.PAT` was used in an `if:` clause here: https://github.com/Open-MSS/MSS/blob/177bef6cdd1d9dd92c32f8d1be1bf9b795f2054a/.github/workflows/testing.yml#L42-L46

I am pretty sure that no other step implicitly relies (or even should rely) on the `PAT` environment variable to be set and there is no longer an explicit reference to `env.PAT`, so I think it can be removed.